Source/ThirdParty/libwebrtc/libwebrtc.xcodeproj/project.pbxproj

```diff
@@ -27278,19 +27312,14 @@
 		};
 		449467C32AE05DA200A9FED0 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			target = FB39D0D01200F0E300088E69 /* libwebrtc */;
+			target = FB39D0D01200F0E300088E69 /* libwebrtc.dylib */;
 			targetProxy = 449467C42AE05DA200A9FED0 /* PBXContainerItemProxy */;
 		};
 		449467D32AE05DD200A9FED0 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
 			target = 449467C22AE05DA200A9FED0 /* rtp_depacketizer_av1_assemble_frame_fuzzer */;
 			targetProxy = 449467D22AE05DD200A9FED0 /* PBXContainerItemProxy */;
 		};
-		449CF1612ADEDE9B00F22CAF /* PBXTargetDependency */ = {
-			isa = PBXTargetDependency;
-			target = 44ABBE882AC63F0C006B44DD /* sdp_integration_fuzzer */;
-			targetProxy = 449CF1602ADEDE9B00F22CAF /* PBXContainerItemProxy */;
-		};
```

Comment:
Is there a way we can be clever and disable building `sdp_integration_fuzzer` when building with automation, but leave `sdp_integration_fuzzer` enabled when building at desk for engineers?

Maybe this isn't a common enough use case to do this, though.

We should have a bug or radar tracking the fix, though.  (We probably do, but it's late, and I'm too tired to look it up.)

Reply:
I think this is not common enough to do. My hope is also that this is short-lived.

Tracked in https://bugs.webkit.org/show_bug.cgi?id=287154, which I'll be mentioning in the new commit message.